model-archiver/model_archiver/model_packaging_utils.py

```diff
@@ -164,6 +165,10 @@ def copy_artifacts(model_name, **kwargs):
 
                 if file_type == "extra_files":
                     for path_or_wildcard in path.split(","):
+                        if not Path(path_or_wildcard).exists():
```

Comment:
this "if" lead [wildcard feature](https://github.com/pytorch/serve/pull/2142) invalid @mreso

Reply:
good catch @otakudj created #3304 to fix